aioseop_class.php

```diff
@@ -1022,48 +1022,54 @@ function custom_output_option( $buf, $args ) {
 	}
 
 	/**
-	 * Get Title Format
+	 * The get_preview_snippet_title() function.
 	 *
-	 * Get Title Format for snippet preview.
-	 * Get the title formatted according to AIOSEOP %shortcodes% specifically for the snippet preview..
+	 * Processes the title format for the snippet preview on the Edit screen.
 	 *
 	 * @since 2.4.9
+	 * @since 3.2.0 Fix #1408 & #2526.
 	 *
 	 * @param array $args
 	 * @return mixed
 	 */
-	public function get_title_format( $args ) {
+	public function get_preview_snippet_title() {
 		$info         = $this->get_page_snippet_info();
 		$title        = $info['title'];
 		$description  = $info['description'];
 		$keywords     = $info['keywords'];
 		$url          = $info['url'];
 		$title_format = $info['title_format'];
 		$category     = $info['category'];
-		$w            = $info['w'];
-		$p            = $info['p'];
+		$wp_query     = $info['w'];
+		$post         = $info['p'];
```

Comment:
Thank you for changing the names on these. This is much better!

Reply:
Thanks, I agree! I originally had no idea what I was looking at and had to browse through the code to find what these variables are.